sonic_platform_base/sonic_sfp/sfputilbase.py

```diff
@@ -954,6 +968,11 @@ def get_transceiver_info_dict(self, port_num):
                         compliance_code_dict[key] = sfp_interface_bulk_data['data']['Specification compliance']['value'][key]['value']
                 transceiver_info_dict['specification_compliance'] = str(compliance_code_dict)
 
+                for key in qsfp_dom_capability_tup:
+                    if key in sfp_dom_capability_data['data']:
+                        dom_capability_dict[key] = "yes" if sfp_dom_capability_data['data'][key]['value'] == 'on' else "no"
+                transceiver_info_dict['dom_capability'] = str(dom_capability_dict)
```

Comment:
Is it necessary to expose this to transceiver_info_dict? My understanding is that this is only some internal flag, if DOM supported, means can get the related meaningful monitor info from the EEPROM.

Reply:
This is a verification method for the SFP type transceiver for now.
If user buy a SFP type transceiver with DOM capability, there is no way to check DOM capability of the transceiver in SONIC currently. With this, the user can check if he/she buy a fake one or not at least.
And in the future, we can enhance this part to check more dom info according to its dom_capability.